firebase.go

```diff
@@ -126,8 +130,23 @@ func (fb *Firebase) doRequest(method string, body []byte) ([]byte, error) {
 	}
 
 	resp, err := fb.client.Do(req)
-	if err != nil {
+	switch err := err.(type) {
+	default:
 		return nil, err
+	case nil:
+		// carry on
+	case *_url.Error:
```

Comment:
> what's the use of having this package name _?

I had put that in @Tonkpils . The intent was exactly what @MrGossett said, so that it wouldn't clash with `url` variables